azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/BatchAccount.java

```diff
@@ -66,13 +64,11 @@
     /**
      * @return the access keys for this batch account
      */
-    @LangMethodDefinition(AsType = LangMethodType.Method)
     BatchAccountKeys keys();
```

Comment:
i noticed that there is also the method `synchronizeAutoStorageKeys()`. Are all these "keys" methods applicable to what that method calls "AutoStorageKeys"?  If so, then i'd think all methods should refer to these keys the same way, i.e.:
`.autoStorageKeys()` instead of `keys()`
`refreshAutoStorageKeys()` instead of `refreshKeys()`